src/test/java/teammates/test/cases/action/InstructorFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -35,15 +35,10 @@ protected String getActionUri() {
     }
 
     @Override
-    protected void prepareTestData() {
-        dataBundle = loadDataBundle("/InstructorFeedbackSubmissionEditSaveActionTest.json");
-        removeAndRestoreDataBundle(dataBundle);
-    }
-
-    @Override
     @Test
     public void testExecuteAndPostProcess() {
         dataBundle = loadDataBundle("/InstructorFeedbackSubmissionEditSaveActionTest.json");
+        removeAndRestoreDataBundle(dataBundle);
```

Comment:
In this `testExecuteAndPostProcess` uses different data bundle instead of typical data bundle